src/uptane_generator/repo.cc

```diff
@@ -221,6 +222,27 @@ void Repo::generateRepo(KeyType key_type) {
   if (repo_type_ == Uptane::RepositoryType::Director()) {
     Utils::writeFile(path_ / DirectorRepo::dir / "manifest", std::string());  // just empty file to work with put method
   }
+
+  generateCampaigns();
```

Comment:
It looks like this will cause a campaign to get generated for every user of uptane-generator, regardless of whether they want it. I don't think that's ideal. If we want to create campaign metadata, we need a separate command for it.

Reply:
Agree. I will fix it.